.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+fail_fast: true
+minimum_pre_commit_version: '0'
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    -   id: check-added-large-files
+    -   id: check-case-conflict
+    -   id: check-json
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: requirements-txt-fixer
+    -   id: trailing-whitespace
+    -   id: no-commit-to-branch
```

Comment:
is there a way to set the `protected` branch in github?

Reply:
Yes there is. You can create a branch protection rule (brazil ws use --package SunGatePython). In that case this hook may be redundant